indent/crystal.vim

```diff
@@ -1,9 +1,26 @@
+let s:cpo_save = &cpo
+set cpo&vim
+
+" Initialization {{{1
+" ==============
+
 " Only load this indent file when no other was loaded.
 if exists('b:did_indent')
   finish
 endif
+
 let b:did_indent = 1
 
+if !exists('g:crystal_indent_assignment_style')
+  " Possible values: 'variable', 'hanging'
+  let g:crystal_indent_assignment_style = 'hanging'
```

Comment:
Would you check behavior of `crystal tool format` for default value of this variable? I guess behavior of `crystal tool format` is "variable".

Reply:
According to my tests, it appears that default values inherited from vim-ruby are the same as `crystal tool format`, so I don't think any change is necessary here.

Reply:
I see. Let's keep 'hanging' as default value.